### ISSUE TITLE
Error: Module version mismatch. Expected 48, got 49.

### ISSUE THREAD

**Issue by jsantell** [mozilla/tofino#892]
Something in 6c422b3b98ea63567e878d84a5dcd6c446fba38a causes this error to be thrown during build. No noticible side effects other than the error message, looks like something with dtrace?

```
Error: Module version mismatch. Expected 48, got 49.
    at Error (native)
    at Object.Module._extensions..node (module.js:568:18)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/jsantell/Dev/tofino/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:541:32)
    at Module._extensions..js (module.js:550:10)
{ Error: Cannot find module './build/default/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/jsantell/Dev/tofino/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:541:32)
    at Module._extensions..js (module.js:550:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jsantell/Dev/tofino/node_modules/babel-register/lib/node.js:156:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module './build/Debug/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/jsantell/Dev/tofino/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:541:32)
    at Module._extensions..js (module.js:550:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jsantell/Dev/tofino/node_modules/babel-register/lib/node.js:156:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12) code: 'MODULE_NOT_FOUND' }
```

**Comment by jsantell** [comment]
Possibly OSX only

**Comment by victorporof** [comment]
CI is permafailing: https://travis-ci.org/mozilla/tofino/jobs/147916761

**Comment by victorporof** [comment]
https://github.com/mozilla/tofino/pull/890

**Comment by Mossop** [comment]
Solved for now by backing out #890
